### PR TITLE
Subarray with palindromic concatenation added.

### PR DESCRIPTION
**Pull request by neha07kumari**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

For a given array and an integer K, we need to check if any subarray of size K exists in the array such that concatenation of elements form a palindrome.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues :
Solves issue #7355

### CODE REVIEW

**Comment by neha07kumari** [comment]
@bhav09  @ankitaggarwal23  please review it and let me know if there are any conflicts.

File: C-Plus-Plus/ds/subarray_palindrome_conact.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+such that concatenation of elements form a palindrome.
+APPROACH:        SLIDING WINDOW APPROACH
+1.store concatenation of initial k elements.
+2.iterate over the array & start deleting elements from the start and adding elements from the end.
+3.at each step,check if concatenation is a palindrome.
+*/
+
+#include<iostream>
+#include<vector>
+#include<cmath>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/subarray_palindrome_conact.cpp:13]: Use single header file **bits/stdc++**

File: C-Plus-Plus/ds/subarray_palindrome_conact.cpp [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            return j-k+1;
+        }
+    }
+
+    return -1;   //represents that we didn't get any palindrome from array
+}
+
+int main(){
+
+    vector<int> arr ={2,3,5,1,1,5};
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/subarray_palindrome_conact.cpp:59]: Use Dynamic Input

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Do the suggested changes

**Comment by siddharth25pandey** [comment]
@neha07kumari Resolve the merge conflicts

**Review by ankitaggarwal23 — COMMENTED** [review]
Thank you for your valuable time.😊

**Pull request closed without merging**